app/models/user.rb

```diff
@@ -6,11 +6,10 @@ class User < ApplicationRecord
   VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
   validates :email, presence: true, length: { maximum: 255 }, format: { with: VALID_EMAIL_REGEX }, uniqueness: { case_sensitive: false }
   has_secure_password
-  validates :password, presence: true, length: { minimum: 6 }, allow_nil: true
```

Comment:
We need this `allow_nil: true` for when we edit the user (and they don't update their password), and for some of the specs which rely on the user already existing with only a digested password stored.